### PR TITLE
update the version messages to 7.34e

### PR DESCRIPTION
**Pull request by ArcherWayne**
update the version messages to 7.34e

### CODE REVIEW

File: docs/changelog/changelog_en.txt [hunk 1]
```diff
@@ -460,4 +460,10 @@ Update to 7.33e
 2. Improved item purchasing system to sell items when inventory is full.
 3. Fix runtime error in buyback.
 4. Fix minor issues in purchasing and courier.
-5. Improve the farming methods of some heroes.
\ No newline at end of file
+5. Improve the farming methods of some heroes.
+v1.7.16(2023.12.03)
+1. Update the builds of some heros to 7.34e.
```
Comment (adamqqqplay) [inline docs/changelog/changelog_en.txt:465]: typo: heroes

File: util/ChatSystem.lua [hunk 2]
```diff
@@ -1,8 +1,8 @@
 local BotsInit = require("game/botsinit")
 local M = BotsInit.CreateGeneric()
 
-local version = "1.7.15"
-local updateDate = "August 20, 2023"
+local version = "1.7.16"
+local updateDate = "December 05, 2023"
```
Comment (adamqqqplay) [inline util/ChatSystem.lua:5]: Need to update release date.

File: docs/changelog/changelog_zh_cn.txt [hunk 3]
```diff
@@ -463,4 +463,10 @@ v1.7.15(2023.08.20)
 2. 改进了物品购买系统，库存满时可以出售物品。
 3. 修复买活时的报错。
 4. 修复物品购买和信使运送中的小问题。
-5. 改进部分英雄的打野方式。
\ No newline at end of file
+5. 改进部分英雄的打野方式。
+v1.7.16(2023.12.03)
```
Comment (adamqqqplay) [inline docs/changelog/changelog_zh_cn.txt:467]: Need to update release date.

File: docs/changelog/changelog_en.txt [hunk 4]
```diff
@@ -460,4 +460,10 @@ Update to 7.33e
 2. Improved item purchasing system to sell items when inventory is full.
 3. Fix runtime error in buyback.
 4. Fix minor issues in purchasing and courier.
-5. Improve the farming methods of some heroes.
\ No newline at end of file
+5. Improve the farming methods of some heroes.
+v1.7.16(2023.12.03)
```
Comment (adamqqqplay) [inline docs/changelog/changelog_en.txt:464]: Need to update release date.

File: docs/changelog/changelog_zh_cn.txt [hunk 5]
```diff
@@ -463,4 +463,10 @@ v1.7.15(2023.08.20)
...
+1. 部分英雄加点和出装更新至7.34e
+2. 修复半人马行者和巨牙海民的技能施放问题.
+3. 修复食人魔魔法师的技能施放问题. 
+4. 优化道具购买.
+5. 修复部分报错.
```
Comment (adamqqqplay) [inline docs/changelog/changelog_zh_cn.txt:472]: Use Chinese period.

**Review by adamqqqplay — COMMENTED** [review]
The commit message needs to be:: "Prepare v1.7.16 release"

**Comment by ArcherWayne** [comment]
sure thing, the issues are fixed.

**Pull request merged**